libraries/provider_hollandbackup_main.rb

```diff
@@ -0,0 +1,27 @@
+require 'chef/provider/lwrp_base'
+require_relative 'helpers'
+
+class Chef
+  class Provider
+    class HollandbackupMain < Chef::Provider::LWRPBase
```

Comment:
It looks like you have many of these providers and resources defined twice -- once under `libraries/...` and once in `resources/...` & `providers/...`. They really should just be in one of those places, especially if they're identical.